python/lib/dependabot/python/file_parser.rb

```diff
@@ -338,7 +338,7 @@ def marker_satisfied?(marker, python_version)
       end
 
       def evaluate_condition(condition, python_version)
-        operator, version = condition.match(/([<>=!]=?)\s*"?([\d.]+)"?/).captures
+        operator, version = condition.match(/([<>=!]=?)\s*"?([\d.]+)"?/)&.captures
```

Comment:
nit, and unrelated to your changes, but a descriptive constant name might help explain the regex here

Reply:
I agree but that can be addressed when we work on this issue: https://github.com/github/dependabot-updates/issues/8099